ztag/schema.py

```diff
@@ -106,6 +106,31 @@ class CensysString(WhitespaceAnalyzedString):
     "timestamp":DateTime(),
 })
 
+ztag_dsa_params = SubRecord({
+    "p":IndexedBinary(),
+    "q":IndexedBinary(),
+    "g":IndexedBinary(),
+    "y":IndexedBinary(),
+})
+
+ztag_ecdsa_public_key = SubRecord({
```

Comment:
Given that the definition is not SSH-specific, it just happens to be used only by SSH right now, I'm not sure that really makes sense. (I've also added `ztag_ed25519_public_key` below, and it's only used by SSH, but I don't think we should add SSH to its name; they both could trivially be reused elsewhere when needed.)